Sources/_TestSupport/DoubleWidth.swift

```diff
@@ -79,10 +90,18 @@ public struct DoubleWidth<Base : FixedWidthInteger> {
   //
   // For that reason, we'll include an internal initializer that takes two
   // separate arguments.
+  
+  /// Creates a new instance from the given tuple of high and low parts.
```

Comment:
Technically, no longer takes a tuple since it’s splatted. Also internal (at least at present).

Reply:
Doc comments are perfectly useful on internal API =)